### DISCUSSION TITLE
Why this error happened while pay the request? Do you have the solution?

### DISCUSSION THREAD

**Discussion by Johnson-8844** [RequestNetwork/requestNetwork#1497] [Questions]
Error: Error: cannot estimate gas; transaction may fail or may require manual gas limit [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (reason="execution reverted: transferFrom() has been reverted", method="estimateGas", transaction={"from":"0xFf43E33C40276FEEff426C5448cF3AD9df6b5741","to":"0x399F5EE127ce7432E4921a61b8CF52b0af52cbfE","data":"0xc219a14d0000000000000000000000000ec435037161acd3bb94eb8df5bc269f17a4e1b9000000000000000000000000ff43e33c40276feeff426c5448cf3ad9df6b574100000000000000000000000000000000000000000000000000000000000f424000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008b6a88d162f8b5cbe000000000000000000000000000000000000000000000000","accessList":null}, error={"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":3,"message":"execution reverted: transferFrom() has been reverted","data":"0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000207472616e7366657246726f6d282920686173206265656e207265766572746564","cause":null},"stack":"{\n  \"code\": -32603,\n  \"message\": \"Internal JSON-RPC error.\",\n  \"data\": {\n    \"code\": 3,\n    \"message\": \"execution reverted: transferFrom() has been reverted\",\n    \"data\": \"0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000207472616e7366657246726f6d282920686173206265656e207265766572746564\",\n    \"cause\": null\n  },\n  \"stack\": \"Error: Internal JSON-RPC error.\\n    at new o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-1.js:1:215217)\\n    at a (

**Comment by aimen74** [comment]
@Johnson-8844 could you please share the code snippet you're using to create a request.
